docs/source/getting_started/getting_started.md

```diff
@@ -106,25 +116,30 @@ Please note that if an interaction between a user and a document is added, the d
 Let's ask for personalised documents again now:
 
 ```bash
-curl https://<url>/users/u1234/personalized_documents?include_properties=true
-    -H "authorizationToken: <front_office_token>"
+curl -X POST "$URL/users/u1234/personalized_documents" \
+    -H "Content-Type: application/json" \
+    -H "authorizationToken: $FRONTOFFICE_TOKEN" \
+    -d '{
+        "include_properties": true
+    }'
```

Comment:
there is `--json` which is `-H "Content-Type: application/json"` + `-d`

Reply:
We are not using it to clarify that you need to set the header...